plugins/woocommerce/.gitattributes

```diff
@@ -1,20 +0,0 @@
-# Set the default behavior, in case people don't have `core.autocrlf` set.
-* text=auto
-
-# Declare files that will always have LF line endings on checkout.
-*.php text eol=lf
-
-# Remove files for archives generated using `git archive`.
-/.*                export-ignore
-bin                export-ignore
-CODE_OF_CONDUCT.md export-ignore
-changelog.txt      export-ignore
-composer.*         export-ignore
-Gruntfile.js       export-ignore
-package.json       export-ignore
-package-lock.json  export-ignore
-phpcs.xml          export-ignore
-phpunit.*          export-ignore
-README.md          export-ignore
-renovate.json      export-ignore
-tests              export-ignore
```

Comment:
Is it safe to remove these `export-ignore` lines? Some tooling might use `git archive` under the hood.

Reply:
Given that it's possible, I've gone ahead and updated this file list to reflect the current state of the `plugins/woocommerce` directory.